src/main/java/com/hackathonorganizer/hackathonwriteservice/hackathon/exception/ErrorResponse.java

```diff
@@ -0,0 +1,19 @@
+package com.hackathonorganizer.hackathonwriteservice.hackathon.exception;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+public class ErrorResponse {
```

Comment:
To moze byc rekordem, proponuje zrobic tak jak ma jakub
public record ErrorMessage(String httpStatus, String message, LocalDateTime timeStamp) {
}